CRUD-APP/user/AddUser.js

```diff
@@ -0,0 +1,89 @@
+import React, {useState} from 'react';
+import axios from 'axios'
+import { useHistory } from "react-router-dom";
+
+const AddUser = () => {
+  let history = useHistory();
+  const [user, setUser] = useState({
+    name: "",
+    username: "",
+    email: "",  
+    phone: "",
+    website: ""
+  });
+
+  const { name, username, email, phone, website } = user;
+  const onInputChange = e => {
+    setUser({ ...user, [e.target.name]: e.target.value });
+  };
+
+  const onSubmit = async e => {
+    e.preventDefault();
+    await axios.post("http://localhost:3003/users", user);
+    history.push("/");
+  };
+ 
+     return(
+        <div className="container">
+        <div className="w-75 mx-auto shadow p-5">
+          <h2 className="text-center mb-4">Add A User</h2>
+          <form onSubmit={e => onSubmit(e)}>

```

Comment:
No need to write function call this way. event will be passed automatically to function argument.
```suggestion
          <form onSubmit={onSubmit}>
```